### PR TITLE
BREAKING chore: use hashing lib and default IPFSv1 hashing for all entitites

### PR DESCRIPTION
**Pull request by menduz**
This may be a breaking change in behavior but catalysts are supposed to accept the changes right now.

It uses the new `@dcl/hashing` lib and generates _**all**_ entities with CIDv1 hashes, deprecating the former `Qm` ad-hoc hashing.

The based on the "usage in the wild" the function `buildEntityAndFile` was moved to catalyst-client https://github.com/decentraland/catalyst-client/pull/198

### CODE REVIEW

**Review by guidota — APPROVED** [review]
LGTM 👍

File: src/utils/Hashing.ts [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-import ipfsHashing from 'ipfs-only-hash'
-import { ContentFileHash, EntityContentItemReference, EntityMetadata } from '../types'
+import * as hashing from '@dcl/hashing'
+import { ContentFileHash } from '../types'
 
 export namespace Hashing {
   /**
    * Given a set of files, return a map with their hash
-   * @deprecated use calculateIPFSHashes instead
+   * @deprecated use calculateIPFSHashes instead. This function only exists for compatibility reasons
```
Comment (agusaldasoro) [inline src/utils/Hashing.ts:7]: shouldn't we remove this method?

File: src/utils/Hashing.ts [hunk 2]
```diff
@@ -20,76 +18,43 @@ export namespace Hashing {
 
   /**
    * Return the given buffer's hash
-   * @deprecated use calculateIPFSHash instead
+   * @deprecated use calculateIPFSHash instead. This function only exists for compatibility reasons
```
Comment (agusaldasoro) [inline src/utils/Hashing.ts:21]: this too?

File: src/utils/Hashing.ts (hunk 1 above)
Comment (menduz) [reply src/utils/Hashing.ts:7]: Yes, I wasn't planning to do it but I will take the chance of the breaking change to do it. On it

**Pull request merged**